mozjs/Cargo.toml

```diff
@@ -12,6 +12,7 @@ doctest = false
 
 [features]
 debugmozjs = ['mozjs_sys/debugmozjs']
+jit = ['mozjs_sys/jit']
```

Comment:
This should be a default feature for the mozjs crate as well.

Reply:
I moved the default feature to mozjs and removed it as default feature from mozjs_sys. Otherwise we couldn't disable it, unless `mozjs` would depend on `monz_js_sys` with `default-features = false`. Since I don't think there would be any other users of the sys crate, besides `mozjs`, I think it makes the most sense to expose the jit feature as default in `mozjs` and don't add any default features to mozjs_sys. 
What do you think?